pkg/reconciler/brokercell/broker_targets_config.go

```diff
@@ -54,6 +55,9 @@ func (r *Reconciler) reconcileConfig(ctx context.Context, bc *intv1alpha1.Broker
 	// maintaining 2 queues for updated brokers and triggers, and only update the config for updated brokers/triggers.
 	brokerTargets := memory.NewEmptyTargets()
 	for _, broker := range brokers {
+		if !utils.BrokerClassFilter(broker) {
```

Comment:
I cannot find the definition of `BrokerClassFilter`, maybe I'm not familiar with this part of the code. can you point to me where this is?

Reply:
sry... just added the new file.